src/reconstruction/modelGeneration/modelProperties/plotModelConsistency.m

```diff
@@ -0,0 +1,241 @@
+function plotModelConsistency(modelResults,modelMetaData,schematicFlag,nRows,nCols,resultsDirectory,resultsFileName)
+%plots stoichoiometric and flux consistency figures, given a modelResults
+%structure or by loading the modelResults structure from a specified
+%location
+%
+%INPUT
+% modelResults          output of checkModelProperties.m
+% modelMetaData         Cell array, where each row is metadata for one model
+%                       with five columns: species, modelID, fileName, PMID, doi.
+% schematicFlag         top corner plot illustrating the different colours
+% nRows                 number of rows in the subplot
+% nCols                 number of rows in the subplot (nRows*nCols >= length(modelResults)
+%
+%OPTIONAL INPUT
+% resultsDirectory      directory where output of checkModelProperties.m has been saved
+% resultsFileName       filename where output of checkModelProperties.m has been saved
+
```

Comment:
Here and elsewhere:
Could you please format the documentation as described in the [documentation guide](https://opencobra.github.io/cobratoolbox/docs/documentationGuide.html)